src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -168,135 +163,94 @@ class NativeAppManager(SqlExecutionMixin):
 
     def __init__(self, project_definition: NativeApp, project_root: Path):
         super().__init__()
-        self._project_root = project_root
-        self._project_definition = project_definition
+        self._project = NativeAppProjectModel(
```

Comment:
nit: Since we're renaming some stuff, we can make this explicit that this is not the entire project, but only the `na_project`